app/controllers/books_controller.rb

```diff
@@ -0,0 +1,12 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+  def show
+    @book = Book.find(params[:id])
```

Comment:
Careful here, if the Book is not found because the id is invalid, this will fail and raise an exception.

You should handle that exception and return a proper response. If the book is not found, the API should return a not found status code, and a message saying the product was not found